int64 bob
---
int64 n_goal
int64 n_deleted
